_data/packages.yml

```diff
@@ -314,17 +353,18 @@
       name: Jonny Saunders
       github_username: sneakers-the-rat
   eic:
-      name: Isabel Zimmerman
+      name:
       github_username: isabelizimm
   reviewers:
-    - name: Phil Dong
+    - name:
```

Comment:
Similar to above, phils name is NOT in their profile so it's erasing likely because of that. instead we want to check the file and if a name is there, use the name from the file. this allows people to update their names here.

Reply:
Update: i'll open an issue based on this comment. 

Update the Contributing file for this repo (or the readme) to explain that we should ONLY update a person's name in the contributors.yml file. If we try to update the packages.yml file with the name it will be overwritten. This is because our workflow treats the contributors.yml file as the singel source of truth and the single plae to manage people. The names listed here assume that we can gather metadata based on a contributor's GitHub username from the contributor file. 

takeaway: update names in the contributors.yml file always (and document this well)

Reply:
No worries! I think other people were also editing names in this packages.yml file. So, really, it's on me for not doing a better job documenting how this workflow works!  I had forgotten myself how things were setup! 

a lot of these items are fixed now however. AND if you want to submit a tiny pr to update names in the contributors.yml file OR if you want to open an issue about it that would work. pyosmeta first opens that file and it will NOT overwrite a manually added name. It will overwrite a missing name if it can find a name associated with the gh username 

there are actually multiple issues and bugs that you've uncovered so it's all good to go through!!